components/email/src/Email.svelte

```diff
@@ -773,6 +773,46 @@
     return fromString;
   }
 
```

Comment:
Participants is now rendered as an array of strings instead a plain string. 
If there's draft in thread, a "**Draft**" string will be added to the top of the array.